linera-core/src/client/mod.rs

```diff
@@ -283,7 +285,7 @@ impl<P, S: Storage + Clone> Client<P, S> {
     pub fn create_chain_client(
         self: &Arc<Self>,
         chain_id: ChainId,
-        known_key_pairs: Vec<AccountSecretKey>,
+        signer: Box<dyn Signer>,
```

Comment:
Since we take an owner in the `Signer` trait, I think that we can hoist this right up into `Client`?  Then we shouldn't need to clone the `Signer` (modulo tests that re-use the signer for different clients).

Reply:
Moved to `Client`.